scss/_base_typography.scss

```diff
@@ -84,6 +84,13 @@
     @extend %x-small-text;
   }
 
+  .p-text--x-small-capitalized {
+    @extend %x-small-text;
+
+    font-weight: $font-weight-bold;
+    text-transform: uppercase;
```

Comment:
Why not use create a `u-capitalise` class and put the contents in a small element?

Reply:
For the smallest text (the one we use as table headers) two things are needed - capitalisation and a change in font-weight, as the light font becomes too unreadable at that size. So if we introduce u-capitalise, we'd have to ask people to always add u-bold or something similar - which seems asking for trouble as it is easy to forget.